src/app/components/elements/Userpic.jsx

```diff
@@ -1,51 +1,43 @@
 import React, {Component, PropTypes} from 'react';
 import {connect} from 'react-redux';
 import shouldComponentUpdate from 'app/utils/shouldComponentUpdate';
-import proxifyImageUrl from 'app/utils/ProxifyUrl';
+import { imageProxy } from 'app/utils/ProxifyUrl';
```

Comment:
Following the surrounding code style this should not have spaces

Reply:
A nice stylistic catch. My understanding is that we're using Airbnb style guidelines, which indicate _everywhere_ should have spaces for module imports of this type. https://github.com/airbnb/javascript#modules
I'll add them for consistency.